### PR TITLE
Add snippet for video effects

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
- Add snippet for video effects
- Update API ref
Screenshot 2024-05-13 at 1 27 45 PM

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3510158

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+};
+
+const videoBackgroundImages = [
+  {
+    key: 'ab1',
+    url: '/assets/backgrounds/contoso.png',
+    tooltipText: 'Custom Background'
+  },
+  {
+    key: 'ab1',
```
Comment (alkwa-msft) [inline packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx:24]: does it make sense the key is the same?

File: packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  groupId: string;
+  formFactor?: 'desktop' | 'mobile';
+  fluentTheme?: PartialTheme | Theme;
+  options?: CallCompositeOptions;
+};
+
+const videoBackgroundImages = [
+  {
+    key: 'ab1',
+    url: '/assets/backgrounds/contoso.png',
```
Comment (PorterNan) [inline packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx:20]: Add a comment saying replace your own background image url after this line

File: packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    tooltipText: 'Custom Background'
+  },
+  {
+    key: 'ab1',
+    url: '/assets/backgrounds/contoso.png',
+    tooltipText: 'Custom Background'
+  }
+];
+export const ContosoCallContainer = (props: ContainerProps): JSX.Element => {
```
Comment (PorterNan) [inline packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx:29]: add a new empty line here

File: packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+import { onResolveVideoEffectDependencyLazy } from '@azure/communication-react';
+import { CallComposite, CallCompositeOptions, useAzureCommunicationCallAdapter } from '@azure/communication-react';
+import { PartialTheme, Stack, Theme } from '@fluentui/react';
+import React, { useMemo } from 'react';
+
+export type ContainerProps = {
+  userId: CommunicationUserIdentifier;
+  token: string;
+  groupId: string;
+  formFactor?: 'desktop' | 'mobile';
```
Comment (PorterNan) [inline packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx:12]: Is formFactor and fluentTheme important for this code snippet? If not let's remove them all togehter

Comment (PorterNan) [reply packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx:12]: Including options - we should provide minimal code to get things work to reduce distraction

**Review by PorterNan — APPROVED** [review]

**Review by edwardlee-msft — APPROVED** [review]

**Pull request merged**